feature-libs/my-account/organization/components/shared/organization-detail/toggle-status-action/toggle-status.component.ts

```diff
@@ -94,7 +95,17 @@ export class ToggleStatusComponent<T extends BaseItem> implements OnDestroy {
   }
 
   protected update(item: T): void {
-    this.itemService.update(item[this.key], this.getPatchedItem(item));
+    this.itemService
+      .update(item[this.key], this.getPatchedItem(item))
+      .pipe(
+        take(1),
```

Comment:
What happens if you move away from this component, i.e. if there's a delay? You'd go in the notify method and quit likely have an error. Probably savest to clean up subscription anyway.

Reply:
I tried with slow 3g mode, disable/enable and go to other page. In if we do that fast, will not see notification and no error, else we can see notification and then we are redirected.

Reply:
I'd probably add an explicit rxjs `delay` of 5000 and see if you face an error or leak.